content/exchange/artifacts/Exchange.Windows.System.PowerShell.DetectResponder

```diff
@@ -0,0 +1,34 @@
+name: Custom.Windows.System.PowerShell.DetectResponder
+author: "Dhruv Majumdar, Jamie Bhoohe"
+description: |
+  This artifact allows to detect responder in the environment
+type: CLIENT_EVENT
```

Comment:
A better description for this artifact would be good - maybe include references? 

https://tcm-sec.com/llmnr-poisoning-and-how-to-prevent-it/